src/PuJetIdSF.cc

```diff
@@ -218,23 +151,10 @@ float PuJetIdSF::getSFError(bool isReal, float pt, float eta)
 std::vector<float> PuJetIdSF::getEvtWeight(bigTree &theBigTree, TLorentzVector tau1, TLorentzVector tau2, std::map<int,double> jets_and_smearFactor, bool cleanJets)
```

Comment:
The cleanJets flag and the corresponding instructions should be removed too.

Reply:
+1

Reply:
done, and updated 2017 skimmer accordingly